packages/luvit/build.sh

```diff
@@ -3,10 +3,11 @@ TERMUX_PKG_DESCRIPTION="Asynchronous I/O for Lua"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION=2.18.1
+TERMUX_PKG_REVISION=2
```

Comment:
You jumped from "0" to "2" :)

Reply:
I thought every packages starts at `1`, or am I mistaken?

Reply:
No, it starts without revision, and we add only if we update package with same version.